levels/roles/sem_people.md

```diff
@@ -0,0 +1,30 @@
+# Senior Engineering Manager - People focus
+
+The 'SEM - People focus' role is concerned with effective and efficient
+recruitment, continuous and professional development and retention of Data
+Engineers and Technical Operations Engineers at The Data Shed. This is achieved
+through the following processes:
+
+## Recruitment
+
+- working closesly with the Talent Aquisition Manager to ensure high quality
+  standards around the recruitment of Data Engineers.
+- ensuring that key skills are recruited for where required.
+
+## Development
+
+- ensuring that Data Engineers are engaged in a stuiable Learning and
+  development plan and have approparite objectives related to it.
+- ensuring Learning and Development days are appropriately planned and utilized.
+- identitfying skill gaps and appropriate Champions that can help reduce those
+  gaps through mentoring and knowledge sharing (utilzing the skills matrix).
```

Comment:
Not sure if you saw my comment on this before it got resolved so just re-iterating. Still missing "i"